sftpretty/__init__.py

```diff
@@ -484,6 +484,9 @@ def get_d(self, remotedir, localdir, callback=None,
             it's st_atime)
         :param bool resume: *Default: False* - Continue a previous transfer
             based on destination path matching.
+        :param int workers: *Default: None* - If None, defaults to number of
+            processors multiplied by 5. Set to less than or equal to allowed
```

Comment:
docs: not sure about the default to be #processeors multiplied by 5. from the doc it says this
```
min(32, (os.cpu_count() or 1) + 4)
```

Reply:
Yeah your right, I should have kept reading. Strange how they just don't update the actual documentation on change and just append another annotation.